ocifs/core.py

```diff
@@ -1274,6 +1274,42 @@ def walk(self, path, maxdepth=None, **kwargs):
             raise ValueError("Cannot crawl all of OCI Object Storage")
         return super().walk(path, maxdepth=maxdepth, **kwargs)
 
+    def glob(self, path, maxdepth=None, **kwargs):
```

Comment:
Shouldn't this be in the ocilake specific implementation file?

Reply:
if the URI contains lake then the customized logic will be executed otherwise fs spec custom implementation will be invoked. This custom logic will not create any impact on the existing use cases